models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
The update_last_login method commits directly to the database, which could cause issues in transactions where multiple updates need to be atomic.